domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/behandling/Førstegangsbehandling.kt

```diff
@@ -49,6 +53,12 @@ data class Førstegangsbehandling(
                 utfallsperioder = emptyList(),
                 status = BehandlingStatus.Manuell,
                 tilstand = BehandlingTilstand.OPPRETTET,
+                kravdatoSaksopplysninger = KravdatoSaksopplysninger(
```

Comment:
Dette er greit for å få inn denne funksjonaliteten. Men du kan godt legge på en kommentar at vi kommer til å trekke denne gradvis inn i Vilkårssett.